src/docs/commands_keybindings.md

```diff
@@ -1,122 +1,141 @@
 ---
-title: Commands and Keybindings
+title: Commands/Menus/Keybindings
 ---
 
-# Commands and Keybindings
+# Commands, Menus and Keybindings
 
-Theia can be extended in a number of different ways. Commands allow packages to contribute unique commands that can then be called by other packages. It's also possible to add keybindings and contexts to these commands, so that they can only be called under certain conditions (window focus, current selection etc.).
+Commands are runnable actions defined by an ID and the function to be executed (plus some optional parameters like a name or an icon). Commands can be triggered via the quick access bar, they can be bound to keybindings or menu items and they can be called programmatically. The action that commands trigger can be context sensitive, so that they can only be called under certain conditions (window focus, current selection etc.).
```

Comment:
minor: we've never referred to the `command palette` or `quick commands menu` as the `quick access bar`, perhaps we should be consistent:

- https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette

Reply:
fixed